### ISSUE TITLE
Could not find a declaration file for module 'elysia-ip'.

### ISSUE THREAD

**Issue by rottmann** [gaurishhs/elysia-ip#5]
The removed `"types": "./src/index.ts",`from root level of package.json in 0.0.6 now raise an error.

**Comment by gaurishhs** [comment]
Surprisingly i got this error on 0.0.5 and to fix it 0.0.6 was released. 

```ts
Could not find a declaration file for module 'elysia-ip'. '/Users/gaurish/test/node_modules/elysia-ip/dist/index.js' implicitly has an 'any' type.
There are types at '/Users/gaurish/test/node_modules/elysia-ip/src/index.ts', but this result could not be resolved when respecting package.json "exports". The 'elysia-ip' library may need to update its package.json or typings.
```

**Comment by gaurishhs** [comment]
I think i should just remove the exports object

**Comment by gaurishhs** [comment]
Can you check and report if the latest commit works, If it does i will create a release.

```sh
bun rm elysia-ip
bun a github:gaurishhs/elysia-ip#6a418d6
cd node_modules/elysia-ip
bun run build
cd ..
```

This will install the latest commit and build that

**Comment by KwaiTushan** [comment]
> Can you check and report if the latest commit works, If it does i will create a release.
> 
> ```shell
> bun rm elysia-ip
> bun a github:gaurishhs/elysia-ip#6a418d6
> cd node_modules/elysia-ip
> bun run build
> cd ..
> ```
> 
> This will install the latest commit and build that. It works for me, the 0.0.6 was a fix for #4 so i assume this one should work probably.

I can confirm it's working

**Comment by gaurishhs** [comment]
@KwaiTushan Thanks for the confirmation, I have published a new release (v0.0.7) containing the fix.
